vserv.py

```diff
@@ -148,7 +155,7 @@ def get_running(self):
         return d
 
     def start_vmx(self, vmx):
-        cmd = [self.vmrun, 'start', vmx]
+        cmd = [self.vmrun, 'start', vmx, 'nogui']
```

Comment:
Another opinionated change here to launch VM's in the background. Would probably be better to have this be a flag